wiki/Rules/it.md

```diff
@@ -0,0 +1,80 @@
+
+Regole di richiesta delle beatmap
+---------------------------------
+
+Queste regole si applicano strettamente alle beatmap inviate tramite il gioco [Beatmap Submission System](/wiki/Beatmap_Submission_System). Questi thread/discussioni sono creati tramite il client di osu! quando una beatmap viene inviata tramite [editor](/wiki/editor).
+
+Esiste una serie di regole più dettagliate per la costruzione di beatmap, note come [Criteri di classificazione](/wiki/Ranking_Criteria).
+
+1. **Assicurati di avere il permesso di usare qualsiasi contenuto coinvolto nella tua beatmap.** Questo include canzoni, video, hit, grafica e qualsiasi altro contenuto che non sia la tua creazione. Se non sei sicuro di dove trovare il contenuto che è libero di essere utilizzato, abbiamo una libreria di brani in crescita disponibile nella scheda [Featured Artist](https://osu.ppy.sh/beatmaps/artists/), tutti al 100% concesso in licenza per l'uso in osu!.
+2. **Rimani in argomento nelle discussioni di Beatmap.** Questi thread/discussioni riguardano la beatmap in questione e nient'altro. Se hai un problema con qualcosa che non appartiene direttamente alla beatmap, pubblicalo nel forum appropriato.
+3. **Non plagiare o tentare di rubare il lavoro degli altri.** Non caricare o utilizzare il lavoro di altre persone senza il loro esplicito permesso (incluse, ma non limitate a, skin e difficoltà degli ospiti).
+
+Regole fuori tema
+---------------
+
+*Le regole seguenti si applicano SOLO al subforum [Off-Topic](https://osu.ppy.sh/community/forums/52), salvo diversa indicazione.*
+
+1. **Nessun argomento in Off-Topic ha valore.** Sono semplicemente, spazzatura. Se i tuoi thread continuano a essere trasferiti in Off-Topic, vengono giudicati indegni di discussioni serie. Accetta il consiglio.
+2. **L'eccessiva stupidità è cattiva.** Essere accattivante stupido va bene, e cade nel modus operandi di shitposting. La vera stupidità non è mai bella. Se non sei in grado di distinguere la differenza, stai già infrangendo la regola.
+3. **I thread di compleanno sono consentiti solo in Off-Topic, e da nessun'altra parte.** Tutte le discussioni di compleanno devono essere accompagnate da non meno di tre (3) immagini di ragazze anime che ballano o saranno *brutalmente* gettate sotto un autobus.
+
+Cosa succede se infrango le regole?
+---------------
+
+Se ti ritrovi nella parte sbagliata delle nostre regole comunitarie, accadranno cose brutte.
+
+Queste cose brutte possono includere (*ma non sono limitate a*):
+
+*La rimozione della tua capacità di chattare nel gioco per un periodo di tempo (noto come silenzio)
+*La rimozione di qualsiasi contenuto illegale dalla tua pagina utente, profilo o beatmap caricati
+*La perdita del privilegio di modificare o alterare la tua pagina utente o i dettagli del profilo
+*La perdita del privilegio di giocare o qualificarsi nei tornei ospitati ufficialmente
```

Comment:
Change the beginning to `-` followed by a space. Change it for this line and the lines above.

e.g 
`*La rimozione della tua capacità di chattare nel gioco per un periodo di tempo (noto come silenzio)` ->
`- La rimozione della tua capacità di chattare nel gioco per un periodo di tempo (noto come silenzio)`